src/Native/build.proj

```diff
@@ -40,7 +40,7 @@
           DependsOnTargets="GenerateVersionSourceFile">
     <PropertyGroup>
       <StripArgs Condition="'$(StripNativeSymbols)' == 'True'">--stripSymbols</StripArgs>
-      <BuildArgs>--configuration $(Configuration) --arch $(TargetArchitecture) $(StripArgs) --mkllibpath $(PackagesDir)mlnetmkldeps/$(MlNetMklDepsPackageVersion)/runtimes/$(PackageRid)/native</BuildArgs>
+      <BuildArgs>--configuration $(Configuration) --arch $(TargetArchitecture) $(StripArgs) --mkllibpath $(PackagesDir)mlnetmkldeps/$(MlNetMklDepsPackageVersion)/runtimes/$(PackageRid)/native --mkllibrpath ../../../../../mlnetmkldeps/$(MlNetMklDepsPackageVersion)/runtimes/$(PackageRid)/native</BuildArgs>
```

Comment:
I just realized that this is wrong.

`mkllibrpath` is used by our customers and we aren't shipping `mlnetmkldeps` to our customers. Instead we are shipping `Microsoft.ML.Mkl.Redist` and should use that package name and Version.

Reply:
(nit) I would toLower `Microsoft.ML.Mkl.Redist`, since NuGet does that.  MacOS typically isn't case sensitive, but it would be best to use the right casing.

---
In reply to: [222124648](https://github.com/dotnet/machinelearning/pull/977#discussion_r222124648) [](ancestors = 222124648)

Reply:
Also, consider adding a comment why we are doing this.

---
In reply to: [222136270](https://github.com/dotnet/machinelearning/pull/977#discussion_r222136270) [](ancestors = 222136270,222124648)